guides/common/modules/proc_installing-the-scc-manager.adoc

```diff
@@ -22,6 +22,7 @@ Perform the following steps to install the _SCC Manager_ plug-in on your {Projec
 ----
 # foreman-rake db:migrate
 # foreman-rake db:seed
+# foreman-rake foreman_scc_manager:setup_authentication_tokens
```

Comment:
Actually, it is only necessary to run the command if you are already running Katello < 4.3 with SCC manager version 1.8.20 and THEN upgrade Katello to 4.3 and up.